src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -622,4 +642,68 @@ public static void updateConfigErrorMessage(Config result, String key, String ms
       }
     }
   }
+
+  public static Map<String, String> getSchemaFromSchemaRegistry(
+      final String topicName, final String schemaRegistryURL) {
+    return getSchemaFromSchemaRegistry(topicName, schemaRegistryURL, "value");
+  }
+
+  public static Map<String, String> getSchemaFromSchemaRegistry(
+      final String topicName, final String schemaRegistryURL, final String type) {
```

Comment:
Can you also put a java doc here? 
Can you also explain what is String type? Why is it hardcoded to "value"

Better name?

Reply:
The schema registry could store schema for either the key and the value (for Kafka). Since most of the time we want the schema for the value, it is set as the default value.

Reply:
Any suggestion for the method name?

Reply:
I see. If we are only doing it for "value", we should explicitly mention it in javadocs. 

Can you start a JIRA for documentation purpose - I think this will be very helpful to announce this feature without looking at the code and finding out what the limitation is. Few I can think of are
1. non - array data type
2. only avro converter
3. Only value converter. 

May be "getValueSchemaFromSchemaRegistry"